core/src/test/scala/unit/kafka/log/LogCleanerParameterizedIntegrationTest.scala

```diff
@@ -325,11 +315,9 @@ class LogCleanerParameterizedIntegrationTest(compressionCodec: String) extends A
 }
 
 object LogCleanerParameterizedIntegrationTest {
-  @Parameters
-  def parameters: java.util.Collection[Array[String]] = {
-    val list = new java.util.ArrayList[Array[String]]()
-    for (codec <- CompressionType.values)
-      list.add(Array(codec.name))
-    list
-  }
+  def all: java.util.stream.Stream[Arguments] =
+    java.util.Arrays.stream(CompressionType.values.map(codec => Arguments.of(codec)))
+
+  def excludeZstd: java.util.stream.Stream[Arguments] =
```

Comment:
Nit: even though this is technically true, it doesn't seem to capture the reason why we exclude zstd. It seems like we basically want to generate the compression types supported by V0 and V1 in this method. That would work better if we add more compression types in the future.

Reply:
I will add some comments for this parameters.